app/views/spree/admin/invoices/_invoices_table.html.haml

```diff
@@ -12,7 +12,7 @@
       - tr_id = spree_dom_id(invoice)
       %tr{:class => tr_class, "data-hook" => "invoice_row", :id => tr_id}
         %td.align-center.created_at
-          = invoice.presenter.invoice_date
+          = pretty_date(invoice.date)
```

Comment:
Now I would argue that `pretty_date` should be called from the presenter, isn't it it's role to format the data ? Then it would make sense to call `invoice.presenter.invoice_date` or maybe something more descriptive like `invoice.presenter.display_date` ? What do you think ?

Reply:
Yes, you're right.